src/perfEvents_linux.cpp

```diff
@@ -594,10 +594,10 @@ int PerfEvents::createForThread(int tid) {
     if (FdTransferClient::hasPeer()) {
         fd = FdTransferClient::requestPerfFd(&tid, &attr);
```

Comment:
`fdtransfer` path is not covered.
`fdtransfer` option allows to offload creation of perf_events to a separate privileged process, see #397 for details.

Reply:
Fixed in 4312d9d48f6906b1c93da8912f4f3641e0c757b4, still need to test it though

Reply:
Tested